src/v/config/configuration.cc

```diff
@@ -480,6 +480,26 @@ configuration::configuration()
        .visibility = visibility::user},
       model::timestamp_type::create_time,
       {model::timestamp_type::create_time, model::timestamp_type::append_time})
+  , log_message_timestamp_alert_before_ms(
+      *this,
+      "log_message_timestamp_alert_before_ms",
+      "Threshold in milliseconds for alerting on messages with a timestamp "
+      "before the broker "
+      "time, as in the past from the point of view of the broker",
+      {.needs_restart = needs_restart::no,
+       .example = "315360000000",
+       .visibility = visibility::tunable},
+      std::chrono::milliseconds::max())
+  , log_message_timestamp_alert_after_ms(
+      *this,
+      "log_message_timestamp_alert_after_ms",
+      "Threshold in milliseconds for alerting on messages with a timestamp "
+      "after the broker "
+      "time, as in the future from the point of view of the broker",
```

Comment:
nit on the descriptions, I find it a little hard to wrap my brain around `as in the future from the point of view of the broker`, possible alternative:

```
Threshold in milliseconds for alerting on messages with a timestamp after the broker's time, meaning they are in the future relative to the broker's clock.
```

Reply:
ye i really need a grammarly plugin for vscode